src/components/Header/notification.js

```diff
@@ -0,0 +1,66 @@
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { Dropdown } from 'react-bootstrap';
+import notificationImg from 'assets/images/notification.png';
```

Comment:
Import our asset after npm packages
In this case, it should be on top of import of NotificationArea

Reply:
done